client/layout/masterbar/style.scss

```diff
@@ -286,3 +286,57 @@ $autobar-height: 20px;
 		margin-right: auto;
 	}
 }
+
+.masterbar__recent-drafts.popover {
+	width: 280px;
+
+	.popover__inner {
+		overflow-y: auto;
+		text-align: left;
+		max-height: 70vh;
+
+		.async-load {
+			margin: 16px;
+			width: auto;
+		}
+	}
+}
+
+.masterbar__publish {
+	display: flex;
+
+	.async-load {
+		display: none;
```

Comment:
Seems a little fragile that we rely on `<Popover />` being rendered outside the current component's DOM tree, otherwise this would cascade and prevent the draft's loader from showing at all. Which raises another question; what in `<MasterbarDrafts />` renders an async component?